spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -413,15 +413,6 @@ def attempt_to_bypass_2fa
     end
   end
 
-  describe 'when the user is not piv/cac enabled' do
-    it 'has no link to piv/cac during login' do
-      user = create(:user, :fully_registered)
-      sign_in_before_2fa(user)
-
-      expect(page).not_to have_link(t('two_factor_authentication.piv_cac_fallback.question'))
-    end
-  end
-
```

Comment:
I think this and the `spec/features/users/sign_in_spec.rb` are another case of `not_to` assertions passing without any actual relevance to the page. I'm assuming earlier iterations of these screens may have had mentions of specific MFAs, rather than leading back to the MFA selection screen via "Choose another authentication method". It only became a problem when we removed the string.